CMakeLists.txt

```diff
@@ -60,6 +60,8 @@ cmake_dependent_option(USE_SSE "Build with x86 SSE instruction support" ON
   "CMAKE_SYSTEM_PROCESSOR STREQUAL x86_64 OR CMAKE_SYSTEM_PROCESSOR STREQUAL amd64" OFF)
 option(USE_F16C "Build with x86 F16C instruction support" ON) # autodetects support if ON
 option(USE_LAPACK "Build with lapack support" ON)
+option(USE_LAPACKE_INTERFACE "Use LAPACKE interface for lapack support" OFF)
```

Comment:
You're later overwriting this option. Why expose it to the user in the first place?

Reply:
I think I can hide this from users. So they can either do 1) build openblas = cblas+lapacke 2) build openblas = cblas and get distro's lapack (not lapacke). Both options are automatically detected by cmake